rectools/visuals/metrics_app.py

```diff
@@ -265,21 +269,19 @@ def _update_chart(
         color_clmn = meta_feature.value if use_meta.value else Columns.Model
 
         # Save dots symbols from the previous widget state
-        # `split(" ", 1)[-1]` removed metainfo from trace name. Thus we guarantee to map with traces from previous state
-        trace_name2symbol = {trace.name.split(" ", 1)[-1]: trace.marker.symbol for trace in self.fig.data}
+        # Remove metainfo from trace name. Thus we guarantee to map with traces from previous state
+        trace_name2symbol = {self._trim_metadata(trace.name)[1]: trace.marker.symbol for trace in self.fig.data}
```

Comment:
Let's call it `model_name2symbol` since it's not full name

Reply:
Done